File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> freq(26);
        for(int i = 0; i < s.size(); i++) {
            freq[s[i] - 'a']++;
        }
        for(int i = 0; i < t.size(); i++) {
            freq[t[i] - 'a']--;
        }
        for(int f: freq) {
            if(f != 0) return false;
        }
        return true;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = Counter(nums)
        heap = []
        for x, cnt in freq.items():
            heapq.heappush(heap, [cnt, x])
            if len(heap) > k:
                heapq.heappop(heap)
        return [num for _, num in heap]
                